soql-stdlib/src/main/scala/com/socrata/soql/functions/SoQLFunctions.scala

```diff
@@ -568,6 +571,10 @@ object SoQLFunctions {
     NoDocs
   )
 
+  val TimeStampAdd = f("timestamp add", FunctionName("date_add"), Map("a" -> TimestampLike, "b" -> Set(SoQLInterval)), Seq(VariableType("a"), VariableType("b")), Seq.empty, VariableType("a"))(
```

Comment:
...`Map("a" -> TimestampLike), Seq(VariableType("a"), FixedType(SoQLInterval))`...

Reply:
Also, any reason this isn't a `SpecialFunctions.Operator("+")` overload?  That should work fine...

Reply:
I get what I want without Interval literals.  I will keep that minimal.

Reply:
i will try +